static/js/publisher/market/publicMetrics.js

```diff
@@ -4,6 +4,7 @@ const NAMES = {
 
 function publicMetrics(form) {
   const publicMetricsEnabled = form['public_metrics_enabled'].checked;
+  const publicMetricsEle = document.querySelector('.js-public-metrics');
```

Comment:
`publicMetricsEle` - why `e` in the end? typo?

Reply:
Just habit - Ele(ment) always done it :D